x-pack/plugins/security_solution/public/detections/components/rules/rule_preview/helpers.ts

```diff
@@ -24,7 +24,7 @@ import { ESQuery } from '../../../../../common/typed_json';
  */
 export const isNoisy = (hits: number, timeframe: Unit): boolean => {
   if (timeframe === 'h') {
-    return hits > 20;
+    return hits > 1;
```

Comment:
Is there a reference somewhere that we're using to decide on these noisy thresholds?

Reply:
Going based off [this comment](https://github.com/elastic/kibana/pull/116374#discussion_r764410980), I think it was the spec in the original feature as well but we should get product to sign off for certain sure

Reply:
a quick [git blame](https://github.com/elastic/kibana/blame/main/x-pack/plugins/security_solution/public/detections/components/rules/rule_preview/helpers.ts#L19-L24) show the 1 alert per hour logic committed 14 months ago